bugzilla/bug.py

```diff
@@ -292,14 +292,20 @@ def addcomment(self, comment, private=False):
 
         return self.bugzilla.update_bugs(self.bug_id, vals)
 
-    def getcomments(self):
+    def get_comments(self):
         """
         Returns an array of comment dictionaries for this bug
         """
         comment_list = self.bugzilla.get_comments([self.bug_id])
         return comment_list['bugs'][str(self.bug_id)]['comments']
 
-
+    def getcomments(self):
+        """
+        Some code bases still use this method. So, for the sake of backward compatibility this method
+        stays in the class' interface.
+        Returns an array of comment dictionaries for this bug just 
+        """
+        return self.get_comments()
```

Comment:
Thank you for the change. If I may make a small nit-picky suggestion to safe a few lines here:

```suggestion
    getcomments = get_comments
```